.install.sh

```diff
@@ -37,6 +37,8 @@ if [ "Windows_NT" != "$OS" ]; then
 else
   # Assumed to be Windows, default to x86
   if [[ "True" == "$IS_64BITS" ]]; then
+    # to build on win 10 with VS 16 use this command:
+    # cmake . -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON -G "Visual Studio 16 2019"
     cmake . -DENABLE_FORMAT=OFF -DBUILD_SHARED_LIBS=ON -G "Visual Studio 14 Win64"
```

Comment:
I think we can use `-A x64` to avoid needing to set a specific version of Visual Studio. (There's an example of this in https://github.com/uber/h3-java/blob/master/src/main/c/h3-java/build-h3-windows.ps1#L69)

Reply:
@isaacbrodsky on my windows setup seems to work properly, I'm adding the change to the PR

Reply:
This change looks good to me, thanks!